src/model/quotes.js

```diff
@@ -115,9 +112,7 @@ class QuotesModel {
         // fail fast on duplicate
         if (dupe.isDuplicateId && (!dupe.isResend)) {
           // same quoteId but a different request, this is an error!
-          throw new Errors.FSPIOPError(quoteRequest,
-            `Quote ${quoteRequest.quoteId} is a duplicate but hashes dont match`, fspiopSource,
-            Errors.ApiErrorCodes.MODIFIED_REQUEST)
+          throw ErrorHandler.CreateFSPIOPErrorFromErrorCode(3106, `Quote ${quoteRequest.quoteId} is a duplicate but hashes dont match`, null, fspiopSource)
```

Comment:
Please use the enum for errorCode from the Error library

Reply:
Done